src/fmu/tools/domainconversion/dconvert.py

```diff
@@ -0,0 +1,596 @@
+from __future__ import annotations
```

Comment:
I mostly skipped reviewing this file assuming that's well and done from sim2seis